.github/ISSUE_TEMPLATE/issue_report.md

```diff
@@ -0,0 +1,10 @@
+<!-- To automatically tag this issue, add the uppercase label(s) surrounded by brackets below, for example: [LABEL] -->
```

Comment:
🤔

Reply:
What exactly is the issue? Emojis aren't really helpful.

Reply:
You don't have auto labeling functionality... read your own PR text?